config.sh

```
@@ -50,3 +50,10 @@ export plugins_list_no_jenkinsfile_main_output_file
 export plugins_list_old_java_main_output_file
 export csv_file_jdk11
 export RATE_LIMIT_DELAY
+
+# New configuration variables for pom.xml analysis
+pom_xml_java_version_xpath="//properties/java.level  | //properties/maven.compiler.source | //properties/maven.compiler.target | //plugin[artifactId='maven-compiler-plugin']/configuration/source | //plugin[artifactId='maven-compiler-plugin']/configuration/target"
```

Shellcheck (0.10.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)